backend/courses/migrations/0013_team_teammember_team_unique_team_name_per_offering.py

```diff
@@ -1,34 +1,41 @@
-# Generated by Django 5.0.7 on 2024-09-27 17:17
+# Generated by Django 5.0.7 on 2024-10-01 22:43
 
 import django.db.models.deletion
+import django.utils.timezone
 import uuid
 from django.db import migrations, models
 
 
 class Migration(migrations.Migration):
 
     dependencies = [
-        ("courses", "0013_team_team_unique_team_name_per_offering"),
+        ("courses", "0012_accommodation_max_grade"),
     ]
 
     operations = [
-        migrations.RemoveField(
-            model_name="team",
-            name="members",
-        ),
-        migrations.AlterField(
-            model_name="enrollment",
-            name="id",
-            field=models.UUIDField(
-                default=uuid.uuid4, editable=False, primary_key=True, serialize=False
-            ),
-        ),
-        migrations.AlterField(
-            model_name="offering",
-            name="id",
-            field=models.UUIDField(
-                default=uuid.uuid4, editable=False, primary_key=True, serialize=False
-            ),
+        migrations.CreateModel(
+            name="Team",
+            fields=[
+                (
+                    "id",
+                    models.UUIDField(
+                        default=uuid.uuid4,
+                        editable=False,
+                        primary_key=True,
+                        serialize=False,
+                    ),
+                ),
+                ("name", models.CharField(max_length=255)),
+                ("created_at", models.DateTimeField(default=django.utils.timezone.now)),
+                (
+                    "offering",
+                    models.ForeignKey(
+                        on_delete=django.db.models.deletion.CASCADE,
```

Comment:
Does this delete offerings if a team is deleted? Team deletion will be expected behavior, lets make sure it nulls. Or does this delete teams on offering deletion (this is ok)